src/value.zig

```diff
@@ -1208,8 +1208,13 @@ pub const Value = extern union {
         };
     }
 
+    /// Write a Value's contents to `buffer`.
+    ///
+    /// Requires that buffer.len >= ty.abiSize(). buffer is allowed to extend past
```

Comment:
s/Requires/Assume: (not sure what formulation you prefer here, the latter part is more like a sentence): 

The buffer is allowed to extend past.